.github/workflows/ecs-deploy-dev.yml

```diff
@@ -5,6 +5,17 @@ on:
   push:
     branches:
       - dev
+    paths-ignore:
+      - 'docs/**'
+      - 'localhost/**'
+      - '.vscode/**'
+      - '.gitignore'
+      - '.github/workflows/mkdocs.yml'
+      - '.github/workflows/pre-commit.yml'
+      - '.pre-commit-config.yaml'
+      - '.pyup.yml'
+      - 'README.md'
+      - 'mkdocs.yml'
```

Comment:
@thekaveman I wasn't sure if I should add *all* non-apps config files like `.flake8`

Reply:
Hmmm, good point @machikoyasuda... this kind of blows up my earlier thought about it being easier to list the files we don't care about 🤣 

Alternatively we can list the things we do care about under `paths` like in #114, which may actually be shorter?

```yaml
- '.aws/**'
- '.github/workflows/ecs-deploy-*.yml'
- 'benefits/**'
- 'bin/**'
- Dockerfile
- gunicorn.conf.py
- nginx.conf
- requirements.txt
```

I'm torn. Either way it's a long list... take the approach that makes the most sense to you 👍

Reply:
The +1 for doing it the same as in #114 (besides being consistent across workflows) is that the explicit `paths` list makes it easier to see what will trigger the workflow, vs. the `paths-ignore` you have to do a bit of mental work to figure out what will trigger.

Reply:
🤷

Reply:
@thekaveman Now that I look at the list of files that **do** matter, it makes more sense to me. This way, it also communicates what files actually matter for the app and deployment. Gonna switch to this strategy!